containers/aesmd/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# This Dockerfile starts the AESM service, which is used by Gramine hardware tests.
+FROM ubuntu:20.04
+
+ARG commit_id
+
+LABEL commit_id=${commit_id}
+
+# Install dependencies
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive  apt-get install -y \
+    wget \
+    gnupg2
+
+RUN wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
+RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' > /etc/apt/sources.list.d/intel-sgx.list
+
+RUN apt-get update
+
+# TODO: Update Conclave to Intel SGX SDK 2.18 and remove package versions: https://r3-cev.atlassian.net/browse/CON-1243
```

Comment:
Can you mention this point in that JIRA ticket so taht it's not lost on whoever picks it up.

Reply:
Done

Reply:
https://r3-cev.atlassian.net/browse/CON-1243